flink-clients/src/test/java/org/apache/flink/client/deployment/ClusterClientServiceLoaderTest.java

```diff
@@ -57,7 +57,7 @@ public void init() {
 	@Test
 	public void testStandaloneClusterClientFactoryDiscovery() {
 		final Configuration config = new Configuration();
-		config.setString(DeploymentOptions.TARGET, StandaloneClientFactory.ID);
+		config.setString(DeploymentOptions.TARGET, StandaloneClientFactory.ID.toUpperCase());
```

Comment:
Isn't using `equalsIgnoreCase` enough?

Reply:
Yes, this is mainly to be "future-proof", i.e. guarantee that the next one who touches this code will respect the contract.